test/tools/libtesteth/TestHelper.cpp

```diff
@@ -181,6 +181,66 @@ set<eth::Network> const& getNetworks()
     return networks;
 }
 
+/// translate network names in expect section field
+/// >Homestead to EIP150, EIP158, Byzantium, ...
+/// <=Homestead to Frontier, Homestead
+set<string> translateNetworks(set<string> const& _networks)
+{
+    // construct vector with test network names in a right order (from Frontier to Homestead ... to
+    // Constantinople)
+    vector<string> forks;
+    for (auto const& net : getNetworks())
+        forks.push_back(test::netIdToString(net));
+
+    set<string> out;
+    for (auto const& net : _networks)
+    {
+        size_t sizeStart = out.size();
+        string possibleNet = net.substr(1, net.length() - 1);
+        vector<string>::iterator it = std::find(forks.begin(), forks.end(), possibleNet);
+
+        if (it != forks.end() && net.size() > 1)
+        {
+            if (net[0] == '>')
+            {
+                while (++it != forks.end())
+                    out.emplace(*it);
+            }
+            else if (net[0] == '<')
+            {
+                while (it != forks.begin())
+                    out.emplace(*(--it));
+            }
+        }
+
+        possibleNet = net.substr(2, net.length() - 2);
+        it = std::find(forks.begin(), forks.end(), possibleNet);
+        if (it != forks.end() && net.size() > 2)
+        {
+            if (net[0] == '>' && net[1] == '=')
+            {
+                while (it != forks.end())
+                    out.emplace(*(it++));
+            }
+            else if (net[0] == '<' && net[1] == '=')
+            {
+                out.emplace(*it);
+                while (it != forks.begin())
+                    out.emplace(*(--it));
+            }
+        }
+
```

Comment:
Perhaps, if `net` is `"ALL"`, that should be translated into all networks.  "ALL" is very similar to ">=Frontier"

Reply:
this would require to refactor a bit more of the other code which rely on ALL

Reply:
also will hawe to change tests that are using ALL

Reply:
I see.  Maybe that's not for now.